src/core.js

```diff
@@ -441,7 +441,6 @@ $.extend( $.validator, {
 			if ( errors ) {
 				// add items to error list and map
 				$.extend( this.errorMap, errors );
-				this.errorList = [];
```

Comment:
Hello, thanks for your interest in my issue. I tested your code and I can tell that it works perfectly! Thank you so much for help. Could you merge this fix in near future?